voting/ethereum/instructions/guardian_registration.md

```diff
@@ -0,0 +1,81 @@
+## Guardian Registration
+In order to participate in the voting, a Guardians must first enroll. 
+An Orbs Guardian is identified and delegated to by its Ethereum address. 
+
+This document walks you through the registration process of Validator and Guardian accounts using the MyCrypto desktop wallet application.
+Any wallet software may be used, and the choice of MyCrypto here is for illustration only.
+
+### A Guardian registration requires to following data:
+- Name and website URL of the enrolling Guardian
+
+Note: A Guardian may update its registration data at any time or leave the registry.
+
+
+### Registration pre-requirements
+ - MyCrypto desktop app (or another equivalent wallet software)
+ - A wallet setup with your Guardian's Ethereum account keys with a positive Ether balance for fees
+ - ABI and contract address available on Etherscan. For example, on Ropsten testnet the contract may be found [here][1] 
+ - 1 Ether deposit required to enroll a Guardian
+ - Registration data
+
+### Enrollment steps
+
+1. **Verify you are on the correct network** ([Choosing the relevant Ethereum Network](./choosing_the_network.md))
+2. Navigate to *Contracts*, under the *Interact* tab.
+![](../instructions/enroll_guardian_1.png)
+1. Enter the `OrbsGuardians` contract address. For Ropsten testnet the address may be found [here][1]. Paste the address in the text 
+box labeled *Contract Address*.
+1. Extract the contract ABI from the Etherscan *code* tab and paste it 
+in the test box labeled *ABI / JSON Interface*. 
+![](../instructions/enroll_guardian_2.png)
+1. Click *Access*.
+* If the *Access* button appears disabled, make sure there are no trailing line feeds at the bottom of the *ABI / JSON Interface* text box.
```

Comment:
* (`*`) here breaks the numbering sequence of the surrounding numbered list of steps. indenting the bullet will fix it (3 spaces).